Kitodo/src/main/java/org/kitodo/production/migration/NewspaperProcessesMigrator.java

```diff
@@ -619,6 +619,22 @@ public void createNextYearProcess() throws ProcessGenerationException, IOExcepti
         }
     }
 
+    private void createWorkpiece(Entry<String, IncludedStructuralElement> yearToCreate, String yearTitle, Process yearProcess) throws IOException {
+        Workpiece yearWorkpiece = new Workpiece();
+        yearWorkpiece.setId(yearProcess.getId().toString());
+        yearWorkpiece.setRootElement(yearToCreate.getValue());
+        RulesetManagementInterface ruleset = ServiceManager.getRulesetService().openRuleset(overallProcess.getRuleset());
+        StructuralElementViewInterface newspaperView = ruleset.getStructuralElementView(yearWorkpiece.getRootElement().getType(), acquisitionStage, Locale.LanguageRange.parse("de"));
+        final Collection<String> processTitleKeys = ruleset.getFunctionalKeys(FunctionalMetadata.PROCESS_TITLE);
```

Comment:
Looks to me like you've messed up something here. `yearTitle` contains the name of the year (e.g. `2021`), but `FunctionalMetadata.PROCESS_TITLE` refers to the metadata fields in which the process title (e.g. `LoreIpsu_1234567X-2021`) is to be written. If you want to save the name of the year it is much easier to get the name of the metadata field: `yearSimpleMetadataView.getId()`.